prow/jobs/kyma-project/runtime-watcher/runtime-watcher.yaml

```
@@ -96,3 +96,61 @@ postsubmits:
           - name: signify-secret
             secret:
               secretName: signify-dev-secret
+    - name: release-runtime-watcher-build
+      annotations:
+        description: "build runtime watcher release image by semantic version tag"
+        owner: "jellyfish"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "watcher-build-release"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-sa-kyma-push-images: "true"
+        preset-signify-prod-secret: "true"
+      always_run: true
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^\d+\.\d+\.\d+(?:-.*)?$
+      spec:
+        containers:
+          - image: "eu.gcr.io/sap-kyma-neighbors-dev/image-builder:v20230313-8dfce5f0b-buildkit"
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            command:
+              - "/image-builder"
+            args:
+              - "--name=runtime-watcher-skr"
+              - "--config=/config/kaniko-build-config.yaml"
+              - "--context=runtime-watcher"
+              - "--dockerfile=Dockerfile"
+              - "--tag=$(PULL_BASE_REF)"
+              - "--export-tags"
+            env:
+              - name: BUILDKITD_FLAGS
+                value: "--oci-worker-no-process-sandbox"
+            resources:
+              requests:
+                memory: 1.5Gi
+                cpu: 1
+            volumeMounts:
+              - name: share
+                mountPath: /home/user/.local/share/buildkit
+              - name: config
+                mountPath: /config
+                readOnly: true
+              - name: signify-secret
+                mountPath: /secret
+                readOnly: true
+        volumes:
+          - name: share
+          - name: config
+            configMap:
+              name: kaniko-build-config
+          - name: signify-secret
+            secret:
+              secretName: signify-dev-secret
```

Signify dev secret is not need.
```suggestion
            volumeMounts:
              - name: share
                mountPath: /home/user/.local/share/buildkit
              - name: config
                mountPath: /config
                readOnly: true
        volumes:
          - name: share
          - name: config
            configMap:
              name: kaniko-build-config
```

---

> Signify dev secret is not need.

I copied this from other teams' configs (e.g. Istio). If they use it for release builds, why it's not needed here?

---

It's not needed in configs of other teams too. This was used in the past as we were signing images with dev instance of signify. At present this is not true anymore and signify dev secret is not used. This is some configuration which was not cleaned when change was applied.